index.html

```diff
@@ -154,6 +154,258 @@ <h2 class="sub-title">Software Developer</h2>
+            <li>html
+              <span class="skill-divider">
+                <svg width="1" height="40" viewBox="0 0 1 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <rect width="1" height="40" fill="#555555" />
+                </svg>
+              </span>
+            </li>
+            <li>Bootstrap
+              <span class="skill-divider">
+                <svg width="1" height="40" viewBox="0 0 1 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <rect width="1" height="40" fill="#555555" />
+                </svg>
+              </span>
+            </li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <div class="card-btn">
+          <a href="" class="btn-text">
+            See Project
+          </a>
+        </div>
+      </div>
+      <div class="project-card">
+        <div class="card-img">
+          <img src="project-img.png" alt="">
+        </div>
+        <div class="card-title">
+          <h4>Multi-Post Stories</h4>
+        </div>
+        <div class="card-text">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+          has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </div>
+        <div class="card-skills-list">
+          <ul>
+            <li>css
+              <span class="skill-divider">
+                <svg width="1" height="40" viewBox="0 0 1 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <rect width="1" height="40" fill="#555555" />
+                </svg>
+              </span>
+            </li>
+            <li>html
+              <span class="skill-divider">
+                <svg width="1" height="40" viewBox="0 0 1 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <rect width="1" height="40" fill="#555555" />
+                </svg>
+              </span>
+            </li>
+            <li>Bootstrap
+              <span class="skill-divider">
+                <svg width="1" height="40" viewBox="0 0 1 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <rect width="1" height="40" fill="#555555" />
+                </svg>
+              </span>
+            </li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <div class="card-btn">
+          <a href="" class="btn-text">
+            See Project
+          </a>
+        </div>
+      </div>
+      <div class="project-card">
+        <div class="card-img">
+          <img src="project-img.png" alt="">
+        </div>
+        <div class="card-title">
+          <h4>Multi-Post Stories</h4>
+        </div>
+        <div class="card-text">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+          has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </div>
```

Comment:
- [ ]  Please consider using a `p` tag here instead of the `div` tag, this is because the content is actually a paragraph and should be put in a paragraph tag `p`. This is to have a semantic HTML workflow.